Dnn.CommunityForums/themes/community-default/templates/TopicsView.ascx

```diff
@@ -49,31 +49,34 @@
+							<td class="dcf-col dcf-col-icon">[FORUMTOPIC:POSTICON|<div><i class="{0}"></i></div>
+                                ]</td>
+							<td class="dcf-col dcf-col-subject" title="[FORUMTOPIC:BODYTITLE]">
 								<div class="dcf-subject">
-										<h4 class="dcf-title dcf-title-4">[SUBJECTLINK][AF:ICONLINK:LASTREAD]</h4>
-										<div class="dcf-topic-started">[RESX:StartedHeader] <i class="fa fa-user fa-blue"></i>&nbsp;[STARTEDBY][AF:UI:MINIPAGER]</div>
+										<h4 class="dcf-title dcf-title-4">[FORUMTOPIC:SUBJECTLINK][FORUMTOPIC:LASTREADURL|<a href="{0}" rel="nofollow"><img src="[FORUM:THEMELOCATION]/images/miniarrow_down.png" style="vertical-align: middle;" alt="[RESX:JumpToLastRead]" border="0" class="afminiarrow" /></a>]</h4>
+										<div class="dcf-topic-started">[RESX:StartedHeader] <i class="fa fa-user fa-blue"></i>&nbsp;[FORUMTOPIC:AUTHORDISPLAYNAMELINK|<a href="{0}" class="af-profile-link" rel="nofollow">[FORUMTOPIC:AUTHORDISPLAYNAME]</a>|[FORUMTOPIC:AUTHORDISPLAYNAME]][AF:UI:MINIPAGER]</div>
 
-									<div class="dcf-forum-description">[BODYTITLE]</div>
+									<div class="dcf-forum-description">[FORUMTOPIC:BODYTITLE]</div>
 
 									<div class="dcf-topic-tools">
-										[ACTIONS:EDIT]
-										[ACTIONS:MOVE]
-										[ACTIONS:LOCK]
-										[ACTIONS:PIN]
-										[ACTIONS:DELETE]
+										[DCF:TOOLBAR:TOPICACTIONS]
 									</div>
 
 									</div>
-										<div>[POSTRATINGDISPLAY]</div>
-										<div rowspan="2" class="dcf-status">[STATUS]</div>
+										<div>[FORUMTOPIC:RATING|<span class="fa-rater fa-rate{0}"><i class="fa fa-star1"></i><i class="fa fa-star2"></i><i class="fa fa-star3"></i><i class="fa fa-star4"></i><i class="fa fa-star5"></i></span>
+]
+										</div>
+										<div rowspan="2" class="dcf-status">[FORUMTOPIC:STATUS]</div>
 
 							</td>
-							<td class="dcf-col dcf-col-replies">[REPLIES]</td>
-							<td class="dcf-col dcf-col-views">[VIEWS]</td>
-							<td class="dcf-col dcf-col-views">[TOPICSUBSCRIBERCOUNT]</td>
-							<td class="dcf-col dcf-col-last-post"><div class="dcf-_lastpost" style="white-space:nowrap;">[LASTPOST][RESX:BY] <i class="fa fa-user fa-blue"></i>&nbsp;[LASTPOSTDISPLAYNAME][AF:ICONLINK:LASTREPLY]<br />[LASTPOSTDATE][/LASTPOST]</div></td>
+							<td class="dcf-col dcf-col-replies">[FORUMTOPIC:REPLYCOUNT]</td>
+							<td class="dcf-col dcf-col-views">[FORUMTOPIC:VIEWCOUNT]</td>
+							<td class="dcf-col dcf-col-views">[FORUMTOPIC:SUBSCRIBERCOUNT]</td>
+							<td class="dcf-col dcf-col-last-post"><div class="dcf-_lastpost" style="white-space:nowrap;">
+                                [LASTPOST]
+                                    [RESX:BY] <i class="fa fa-user fa-blue"></i>&nbsp;[LASTPOSTDISPLAYNAME][FORUMTOPIC:LASTREPLYURL|<a href="{0}" rel="nofollow"><img src="[FORUM:THEMELOCATION]/images/miniarrow_right.png" style="vertical-align: middle;" alt="[RESX:JumpToLastReply]" border="0" class="afminiarrow" /></a>]<br />[FORUMTOPIC:LASTPOSTDATE]
```

Comment:
@johnhenley Shouldn't we avoid adding new `style` attributes to the HTML we're adding, and put it into CSS instead? 🤔 CC: @Timo-Breumelhof